client/my-sites/stats/post-trends/day.jsx

```diff
@@ -29,18 +30,36 @@ class PostTrendsDay extends PureComponent {
 	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	buildTooltipData = () => {
 		const { label, postCount } = this.props;
-		const content = this.props.translate( '%(posts)d post', '%(posts)d posts', {
-			count: postCount,
-			args: {
-				posts: postCount,
-			},
-			comment: 'How many posts published on a certain date.',
+		const date = new Date( label );
+		const weekDay = date.toLocaleDateString( undefined, { weekday: 'long' } );
+		const formattedDate = date.toLocaleDateString( undefined, {
+			month: 'long',
+			day: 'numeric',
+			year: 'numeric',
```

Comment:
Noting that for users whose browser language is different than the Calypso interface language, this might display in a different locale, and can end up being a confusing experience. I believe we must match the Calypso interface language at any given time (cc @Automattic/i18n for tips).

Reply:
Thanks for bringing this up, @tyxla! I saw that in some files we're using `toLocaleDateString( 'en-US', ... )`. Would this be the correct choice here? The other usage of this method that I found on the repo is passing the `locales` param as `undefined` as I did.

Reply:
What @dlind1 suggests makes sense to me, although I'd be happy to blindly trust @Automattic/i18n on a call about what i18n APIs to use.

Reply:
@dlind1 thanks again for helping me with this. This implementation worked fine and I just pushed it.